### DISCUSSION TITLE
Fuzzy logic controller on Crazyflie 2.1

### DISCUSSION THREAD

**Discussion by Vrinda73** [bitcraze/discussions#310] [Q&A] [answered]
Hello everyone!
I am new to this discussion forum. I wanted to implement a fuzzy logic controller on the Crazyflie 2.1 platform. I wanted to know how I can get started. Please can anyone help me regarding this ?
Thank you for your time!

**Comment by knmcguire** [comment, marked as answer]
Hi! This question has already been asked on this forum and the old:
- https://github.com/orgs/bitcraze/discussions/64
- https://forum.bitcraze.io/viewtopic.php?p=24309&hilit=controller#p24309

Please check out the [controller and estimators label](https://github.com/bitcraze/discussions/discussions?discussions_q=label%3A%22Controllers+and+Estimators%22), and if you can still not find the answer your looking for, please let us know.

**Reply by Vrinda73** [reply]
Thank you very much ! Apologies for inconvenience. I will take a look at the documentation. I quickly went through the links you sent me. The problem is that fuzzy controller has its own python library [https://pythonhosted.org/scikit-fuzzy/auto_examples/plot_control_system_advanced.html](url)  I just wanted to know how to interface it with Crazyflie platform. Then I can easily send attitude commands.
